<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PumpParam</class>
 <widget class="QDialog" name="PumpParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2024</width>
    <height>885</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>740</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/confirmbt.png);</string>
   </property>
   <property name="text">
    <string>  保存</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>740</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/back.png);</string>
   </property>
   <property name="text">
    <string>  返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>181</width>
     <height>741</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_pump1_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump2_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump3_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump4_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump5_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump6_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump7_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump8_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump9_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump10_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>684</x>
     <y>10</y>
     <width>191</width>
     <height>771</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="lEdit_pump1">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump2">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump3">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump4">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump5">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump6">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump7">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump8">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump9">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump10">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1014</x>
     <y>21</y>
     <width>261</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_pump11">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>裂解温度1：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump12">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>裂解温度2：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump13">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>裂解温度3：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump14">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>裂解温度4：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump15">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>裂解温度5：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1276</x>
     <y>23</y>
     <width>131</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_pump11_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump12_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump13_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump14_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump15_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>10</y>
     <width>148</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLineEdit" name="lEdit_pump11">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump12">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump13">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump14">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEdit_pump15">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1441</x>
     <y>20</y>
     <width>191</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_pump11_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump12_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump13_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump14_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump15_A">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>调整为：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>21</y>
     <width>281</width>
     <height>741</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_pump1">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>流体参数1：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump2">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>流体参数2：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump3">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>流体参数3：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump4">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>流体参数4：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump5">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>流体参数5：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump6">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>流体参数6：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump7">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>流体参数7：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump8">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>流体参数8：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump9">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>流体参数9：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump10">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>流体参数10：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>20</y>
     <width>181</width>
     <height>741</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_pump1_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump2_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump3_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump4_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump5_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump6_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump7_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump8_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump9_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_pump10_data">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 32, 220);</string>
      </property>
      <property name="text">
       <string>TextLa</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox_machine">
   <property name="geometry">
    <rect>
     <x>1840</x>
     <y>-30</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
